openassessment/assessment/models/base.py

```diff
@@ -552,6 +598,28 @@ def get_median_score(scores):
             )
         return median_score
 
+    @staticmethod
+    def get_mean_score(scores):
+        """Calculate the mean score from a list of scores
+
+        Args:
+            scores (list): A list of int values. These int values
+                are reduced to a single value that represents the mean.
+
+        Returns:
+            (int): The mean score.
+
+        Examples:
+            >>> scores = [5, 6, 12, 16, 22, 53]
+            >>> Assessment.get_mean_score(scores)
+            19
+
+        """
+        total_criterion_scores = len(scores)
+        if total_criterion_scores == 0:
+            return 0
+        return int(math.ceil(sum(scores) / float(total_criterion_scores)))
```

Comment:
`math.ceil` already returns an integer.

```suggestion
        return math.ceil(sum(scores) / float(total_criterion_scores))
```

Reply:
Done. Thank you!